libs/domain/picking/src/components/picking-header/picking-header.component.ts

```diff
@@ -55,12 +73,20 @@ export class PickingHeaderComponent extends PickingListMixin(LitElement) {
           triggerType: 'icon',
           contentBehavior: 'modal',
         }}
-      ></oryx-site-navigation-item>`;
+      ></oryx-site-navigation-item>
+      <oryx-discard-picking
+        ${ref(this.discardModal)}
+        @oryx.back=${this.back}
+      ></oryx-discard-picking>`;
+  }
+
+  protected openDiscardModal(): void {
+    const modal = this.discardModal.value;
```

Comment:
when change ref to query, just:
`this.discardModal?.toggleAttribute('open', true)`